openfga/v1/openfga_service.proto

```diff
@@ -751,12 +751,11 @@ service OpenFGAService {
     };
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
-      summary: "[EXPERIMENTAL] List the users matching the provided filter who have a certain relation to a particular type."
+      summary: "List the users matching the provided filter who have a certain relation to a particular type."
       tags: ["Relationship Queries"]
       operation_id: "ListUsers"
       description:
         "The ListUsers API returns a list of all the users of a specific type that have a relation to a given object.\n "
-        "This API is available in an experimental capacity and can be enabled with the `--experimentals enable-list-users` flag. \n"
```

Comment:
This is slightly premature but the `enable-list-users` experimental option will be removed from OpenFGA as a fast follow to this PR.